docs/commands/zed.md

```diff
@@ -574,6 +570,28 @@ API for automation.
 
 > Note that the branchlog meta-query source is not yet implemented.
 
+### Manage
+```
+zed manage [options]
+```
+Maintenance tasks are performed on a lake with the `manage` command.
```

Comment:
Nit: Most of the other command sections start like this. I think it reads better too.
```suggestion
The `manage` command performs maintenance tasks on a lake.
```